audit/audit.go

```diff
@@ -32,6 +37,28 @@ const (
 	CryptoSignJWSEvent = "SignJWS"
```

Comment:
Why are these constants in not in the crypto package?

Reply:
I chose managability (is that a word?) over code cohesion here, since having them all 1 place makes it much easier to see what events are audited.